code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -858,6 +858,8 @@
 	icon_state = "hair_popcorn"
 	species_allowed = list("Diona")
 	do_colouration = 0
+	gender = FEMALE
+	gender = MALE
```

Comment:
what

Reply:
it looks like it shouldn't work but it does

Reply:
female diona cant use it

Reply:
yes they can

Reply:
this definitely only sets `gender = MALE`, it's the last one that matters